### DISCUSSION TITLE
Evergreen Rates

### DISCUSSION THREAD

**Discussion by MMath19103** [CMSgov/price-transparency-guide#83] [Q&A] [answered]
we have rates that are 'evergreen' so there are no end dates necessarily. Is there a specific requirement concerning this type of contract? would the date be add as 9999-12-31?? 
Thanks

**Comment by ghost** [comment, marked as answer]
Similar to question previously posted as, "If no expiration_date is known, would 9999-12-31 be the appropriate value to represent no defined expiration date? #42"

**Reply by MMath19103** [reply]
Thank you - didn't see that

**Reply by shaselton-usds** [reply]
@BobSyracuse is correct -- similar discussion going on there. Here is the commit that addresses your question: https://github.com/CMSgov/price-transparency-guide/commit/3ca66505da4cf1072d1b4665b0d78699ab26c0e2
